src/m365/tenant/commands/serviceannouncement/serviceannouncement-healthissue-get.ts

```diff
@@ -0,0 +1,64 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  issueId: string;
+}
+
+class TenantServiceAnnouncementHealthissueGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.SERVICEANNOUNCEMENT_HEALTHISSUE_GET;
+  }
+
+  public get description(): string {
+    return 'Gets a specified service health issue for tenant';
+  }
+
+  public defaultProperties(): string[] | undefined {
```

Comment:
`defaultProperties` are meant to be used only on `list` commands where we return multiple items. Since we return a single item here, we shouldn't define them so that we return all properties.